CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## UNRELEASED
+* **IMPROVEMENT** (by @MattiaPispisa) custom border radius in BarTouchTooltipData, #1715
```

Comment:
You don't need to write a changelog here, please read our [contributing guideline](https://github.com/imaNNeo/fl_chart/blob/main/CONTRIBUTING.md)

Reply:
I had read the guidelines, and it was recommended to start from an issue. From the ‘Mention the related issues’ section, what I did seemed correct. What should I have done differently?
Thank you.

Reply:
Ahh, you're right. I'm sorry I forgot to update one part of the guideline.
I just updated it [here](https://github.com/imaNNeo/fl_chart/pull/1775/commits/cef2d74363c4ddda20ab22c075ead9b86fd75ad0).

As we had a lot of conflicts in the change log part, I decided to update it by myself. So you don't need to write something.

Maybe, it would be a good idea to just copy-paste yours in the PR's description. This way I can just add it.